### PR TITLE
Add and update favicons for all devices

### PR DESCRIPTION
**Pull request by Arhell**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | no

### Description

resolve https://github.com/laminas-api-tools/api-tools.getlaminas.org/issues/44

### CODE REVIEW

File: module/Application/view/layout/layout.phtml [hunk 1]
```diff
@@ -6,13 +6,37 @@
...
+            ->appendName('sapplication-config', '/img/favicon/browserconfig.xml')
+            ->appendName('theme-color', '#ffffff')
+            ->appendHttpEquiv('X-UA-Compatible', 'IE=edge')
+    ?>
 
     <?= $this
+            ->headLink([
+                'rel' => 'apple-touch-icon',
+                'sizes' => '180x180',
+                'href' => $this->basePath() . '/img/favicon/apple-touch-icon.png'
```
Comment (froschdesign) [inline module/Application/view/layout/layout.phtml:19]: Please do not add any more logos or favicons to this repository. Everything goes to the repository of the project website: https://github.com/laminas/getlaminas.org

**Review by froschdesign — CHANGES_REQUESTED** [review]

File: module/Application/view/layout/layout.phtml (hunk 1 above)
Comment (froschdesign) [reply module/Application/view/layout/layout.phtml:19]: Related to: https://github.com/laminas/laminas.github.io/pull/176#issuecomment-1479060492

Comment (Arhell) [reply module/Application/view/layout/layout.phtml]: Done

**Comment by weierophinney** [comment]
@froschdesign Is this one ready to go at this point?

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
@Arhell 
Thank you for your time, your patience and this contribution! 👍

**Pull request merged**